### PR TITLE
Updated urls, readme and fixed compose.lock

### PR DESCRIPTION
**Pull request by tolstenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,81 @@
...
+Done, phptest:v3 deployed to Workflow
 
 Use 'deis open' to view this application in your browser
 
 To learn more, use 'deis help' or visit https://deis.com/
 
-To ssh://git@deis-builder.deis.rocks:2222/zanier-zeppelin.git
+To ssh://deis-builder.deis.rocks:2222/phptest.git
  * [new branch]      master -> master
-$ curl http://zanier-zeppelin.deis.rocks
```
Comment (kingdonb) [inline README.md:63]: Could you put these lines back so the documentation example shows the example working?

**Comment by kingdonb** [comment]
Thanks for your contribution!

File: README.md (hunk 1 above)
Comment (tolstenko) [reply README.md:63]: Sure, my bad!

**Review by Cryptophobia — APPROVED** [review]
lgtm

**Pull request merged**